packages/ui/src/lib/Providor.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
```

Comment:
packages/ui/src/lib/Provid**e**r.tsx > packages/ui/src/lib/Provid**o**r.tsx

Also, can we make a dir for it? So it's clear that each component is a folder.

Reply:
> packages/ui/src/lib/Provid**e**r.tsx > packages/ui/src/lib/Provid**o**r.tsx

My bad, I didn't find there is a typo.

> Also, can we make a dir for it? So it's clear that each component is a folder.

Agree, we should do this.